sympy/series/gruntz.py

```diff
@@ -626,9 +625,6 @@ def rewrite(e, Omega, x, wsym):
 
     # Some parts of sympy have difficulty computing series expansions with
     # non-integral exponents. The following heuristic improves the situation:
-    exponent = reduce(ilcm, denominators, 1)
```

Comment:
As you are removing this code, has the situation for which it was originally put here been handled?